### DISCUSSION TITLE
Nextjs / Import a component

### DISCUSSION THREAD

**Discussion by kamalkech** [saadeghi/daisyui#1774] [Q&A] [answered]
### What version of daisyUI are you using?

v2.51.6

### Describe your issue

I create an component Header.tsx in folder src/components/
I import this component in index.tsx but he header not render correctly if put the same code directly in index.tsx render good.

So why using component partial / children, not working ???

![image](https://user-images.githubusercontent.com/1313428/234429164-a74c6ba9-7244-4e1b-b214-e624f99bbd16.png)

### What browsers are you seeing the problem on?

Chrome

**Comment by saadeghi** [comment, marked as answer]
Make sure the path to all your template files (pages, components, etc) is addressed in `tailwind.config.js`:  
https://tailwindcss.com/docs/content-configuration
